spec/functional/resource/registry_spec.rb

```diff
@@ -285,6 +309,30 @@ def create_deletable_keys
         @new_resource.run_action(:create)
         expect(@registry.key_exists?(reg_child + '\Slitheen')).to eq(false)
       end
+      it "does not throw an exception if the action create and type key missing in values hash" do
```

Comment:
since we're actually raising, we should be pedantic and say that it `does not raise an exception` rather than throw.

https://coderwall.com/p/lhkkug/don-t-confuse-ruby-s-throw-statement-with-raise